app/views/groups/_request.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= form_for @group do |f| %>
```

Comment:
For this page, the indentation is messy. Let's be more careful with it in the future.